integration-cli/docker_cli_import_test.go

```diff
@@ -1,32 +1,51 @@
 package main
 
 import (
-	"fmt"
 	"os/exec"
 	"strings"
 	"testing"
 )
 
 func TestImportDisplay(t *testing.T) {
-	server, err := fileServer(map[string]string{
-		"/cirros.tar.gz": "/cirros.tar.gz",
-	})
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "true")
+	out, _, err := runCommandWithOutput(runCmd)
 	if err != nil {
-		t.Fatal(err)
+		t.Fatal("failed to create a container", out, err)
 	}
-	defer server.Close()
-	fileURL := fmt.Sprintf("%s/cirros.tar.gz", server.URL)
-	importCmd := exec.Command(dockerBinary, "import", fileURL, "cirros")
-	out, _, err := runCommandWithOutput(importCmd)
+	cleanedContainerID := stripTrailingCharacters(out)
+	defer deleteContainer(cleanedContainerID)
+
+	exportCmd := exec.Command(dockerBinary, "export", cleanedContainerID)
+	importCmd := exec.Command(dockerBinary, "import", "-")
+	importCmd.Stdin, err = exportCmd.StdoutPipe()
```

Comment:
OK, in that case, I'm sold.  Not at a computer right now, but will update
today.